lib/internal/child_process/helper.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
There appears to be absolutely no reason to separate this out to a different file